README.md

```diff
@@ -2,8 +2,34 @@ For some context on why we think having a code of conduct for our organization i
 
 # DigitalOcean's Engineering Code of Conduct
 
+At DigitalOcean we strive to be a values driven company.
+
+First lets look at those values
```

Comment:
"First lets" => "First, let's"

Reply:
On additional reading I think the line is extraneous so I removed it altogether.